rep-0105.rst

```diff
@@ -81,9 +84,31 @@ The coordinate frame called ``base_link`` is rigidly attached to the
 mobile robot base. The ``base_link`` can be attached to the base in
 any arbitrary position or orientation; for every hardware platform
 there will be a different place on the base that provides an obvious
-point of reference. Note that REP 103 [1] specifies a preferred
+point of reference. Note that REP 103 [1]_ specifies a preferred
 orientation for frames.
 
+earth
+'''''
```

Comment:
In the hopes that ROS systems may some day operate on the Moon or Mars, would some more-generic name make sense? (``global`` or ``planetary``, perhaps?)